fabtests/scripts/runfabtests.sh

```diff
@@ -595,9 +625,9 @@ case ${opt} in
 	;;
 	g) GOOD_ADDR=${OPTARG}
 	;;
-	f) read_exclude_file ${OPTARG}
+	f) exclude_file=${OPTARG}
```

Comment:
This would break specifying multiple exclude files. Makefile.am specifies some OS specific exclude file.

Reply:
Ah -- I'll need to figure out how to support multiple exclude files then.  My bash is the suck, so I missed that that feature was supported.

Reply:
The previous code where we call the function instead of setting a global exclude_file variable supports multiple exclude files.

Reply:
Yeah, I changed that because I wanted to reset the exclusions between running udp and tcp.  I'll redo this area so multiple files can be specified.